lib/Basics/files.cpp

```diff
@@ -50,6 +52,10 @@
 #include "Logger/Logger.h"
 #include "Random/RandomGenerator.h"
 
+#ifdef USE_ENTERPRISE
+#include "Enterprise/Encryption/EncryptionFeature.h"
+#endif
```

Comment:
@matthewvon I'm still troubled by this.

Reply:
@danielhlarkin I am will to create a long-term clean-up issue.  But restructuring, or worse copying, encryption into the base library is too risky and time consuming in my opinion.  I cannot move TRI_Slurp to arangod/arangosh because it is used by lib/V8/v8-utils.cpp to provide test coverage.  I might say "risk not having test coverage" if the tests had not already found one encryption bug with import/export.

Reply:
Agreed.